src/SourceBuild/content/test/Microsoft.DotNet.Tests/PackageSourceMappingsTests.cs

```diff
@@ -0,0 +1,242 @@
+﻿using System.Collections.Generic;
```

Comment:
Hmm this file is missing a license header. I think we are missing a rule in the editorconfig in the VMR.

Reply:
Fixed with https://github.com/dotnet/sdk/pull/44649/commits/f13f4f1cfb5a7adec023571eaf49ddcc8a051648